ocp_resources/benchmark.py

```diff
@@ -0,0 +1,99 @@
+import logging
+
+from openshift.dynamic.exceptions import NotFoundError
+
+from ocp_resources.resource import NamespacedResource
+from ocp_resources.utils import TimeoutSampler
+
+
+LOGGER = logging.getLogger(__name__)
+
+API_TIMEOUT = 30
```

Comment:
These are used only once; please remove. They can be re-added once needed in more than one place.

Reply:
removed